command/base.go

```diff
@@ -40,8 +40,9 @@ type BaseCommand struct {
 	flagTLSSkipVerify bool
 	flagWrapTTL       time.Duration
 
-	flagFormat string
-	flagField  string
+	flagFormat  string
+	flagField   string
+	flagNoColor bool
```

Comment:
I think we can remove this and the var below if we are just using the envvar

Reply:
The idea was to allow it in a flag but not advertise it. Now that you mention autocorrect below, I wonder if it's better to just remove it rather than have people get it autocorrected and then complain that it's not documented. What do you think?

Reply:
I think the code will be a lot simpler if we drop the flag entirely. People are more likely to "never want colored output" that "not want colored output for this specific command", and even in the latter case, they can specify the envvar first.